src/IFrameRoutes.js

```diff
@@ -21,12 +21,12 @@ import {SliderHero} from './SliderCatalog';
 import {SnackbarHero} from './SnackbarCatalog';
 import {SwitchHero} from './SwitchCatalog';
 import {TabsHero} from './TabsCatalog';
-import {TextFieldHero} from './TextFieldCatalog';
+import {TextFieldHeroLegacy} from './TextFieldCatalog';
 import {TopAppBarHero} from './TopAppBarCatalog';
 
 const routesList = [{
   urlPath: 'button',
-  Component: ButtonHero,
+  Component: ButtonHeroLegacy,
```

Comment:
do we need a card and top app bar legacy component as well?

Reply:
No, we just need a Button and TextField because we have experiments running for those component.

Reply:
I'm probably missing context here, but where do these routes apply, given that I'm clearly seeing the new stuff when I browse the catalog normally?

Reply:
The Legacy routes are used for the experiments for material.io. You can get there by adding `iframe` after component but before button. Once the experiment finishes, we'll delete the Legacy routes and components. 

https://material-components.github.io/material-components-web-catalog/#/component/iframe/button